backend/src/platform/api/main.py

```
@@ -41,30 +41,53 @@ def create_app():
+            env_id_index = path_parts.index("env") + 1 if "env" in path_parts else None
+            if env_id_index and env_id_index < len(path_parts):
+                env_id = path_parts[env_id_index]
+                session = sessions.get_session_for_environment(env_id)
+                return {
+                    "request": request,
+                    "session": session,
+                    "environment_id": env_id,
+                    "user_id": None,
+                    "impersonate_email": None,
+                }
+        raise PermissionError("missing environment identifier")
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)